ctwrap/modules/minimal.py

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
 """Minimal example for ctwrap compatible simulation module"""
 
-from ruamel import yaml
-import pandas as pd
 import time
```

Comment:
Let’s omit PEP 484 typing here ... these modules are meant to be as simple as possible, and it may be confusing to others (we want this to be used after all)